src/main/kotlin/onboarding/Problem6.kt

```diff
@@ -1,5 +1,64 @@
 package onboarding
 
-fun solution6(forms: List<List<String>>): List<String> {
-    TODO("프로그램 구현")
+import java.util.regex.Pattern
+
+
+const val DOMAIN = "@email.com"
+
+fun emailLength(email: String): Boolean {
+    return email.length in 11 until 20
+}
+
+fun nickNameLength(nickname: String): Boolean {
```

Comment:
```suggestion
fun nickNameLength(nickname: String): Boolean = nickname.length in 11 until 20
// or
private fun String.nicknameLength: Boolean = this.length in 11 until 20
```
가독성이 좋은 코틀린이니 만큼 조금 더 간략하게 표현할 수 있을 것 같아요

fun nickNameLength가 아니라 fun nicknameLength가 카멜케이스에 맞는 표현 같아요